freqtrade/exchange/exchange.py

```diff
@@ -1528,6 +1529,52 @@ def get_historic_trades(self, pair: str,
             self._async_get_trade_history(pair=pair, since=since,
                                           until=until, from_id=from_id))
 
+    @retrier
+    def get_funding_fees_from_exchange(self, pair: str, since: Union[datetime, int]) -> float:
+        """
+            Returns the sum of all funding fees that were exchanged for a pair within a timeframe
+            :param pair: (e.g. ADA/USDT)
+            :param since: The earliest time of consideration for calculating funding fees,
+                in unix time or as a datetime
+        """
+
+        if not self.exchange_has("fetchFundingHistory"):
+            raise OperationalException(
+                f"fetch_funding_history() has not been implemented on ccxt.{self.name}")
+
+        if type(since) is datetime:
+            since = int(since.strftime('%s'))
```

Comment:
I don't think `%s` (small s) is a valid format string 

It's not listed in the [format codes page](https://docs.python.org/3/library/datetime.html)  ... so this is probably why that test fails in windows (it might pass in linux by accident - but i don't think this is right.

If you require a timestamp - you should instead use `int(since.timestamp())` (eventually multiplied by 1000, depending how you need it).

Reply:
this is one occuance - but not the only one ;)